app.rb

```diff
@@ -6,16 +6,22 @@
 require './teacher'
 require './list_module'
 require './create_module'
+require './storage_module'
 
 class App
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/202222754-7eb1d2a4-cd49-495a-b4f5-c7e585a5c902.png)

**[Optional]**
- [ ] Kindly remove these objects which show up after running your application shown in the image above :arrow_up: